spec/controllers/teams_controller_spec.rb

```diff
@@ -184,7 +184,29 @@
 
             expect(Role.last.user).to eql existing_user
           end
+        end
+
+      #TODO: refactor this test
+      context 'selecting the conference options' do
+        let(:conference_1) { FactoryGirl.create(:conference, :in_current_season)}
+        let(:conference_2) { FactoryGirl.create(:conference, :in_current_season)}
+        let(:team)         { FactoryGirl.create(:team, :in_current_season) }
+        let(:conferences) do
```

Comment:
Personal preferences, but I find the name a bit misleading... How about "team_params"?